pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Run command:
+
+`ansible {{group}} -m command '{{command}}'`
+
+- Run command with sudo:
+
+`ansible {{group}} -bKm command '{{command}}'`
+
+- Ping:
+
+`ansible {{group}} -m ping`
+
+- Get system info:
+
+`ansible {{group}} -m setup`
+
+- List computers belonging to a group:
+
+`ansible {{group}} --list-hosts`
+
+- Using a custom inventory file:
```

Comment:
Sounds good. I'll tweak the descriptions so that they use verbs and and unambiguous